Gems/ROS2/Code/Source/Utilities/ROS2Names.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ROS2Names.h"
+
+namespace ROS2
+{
+    AZStd::string ROS2Names::GetNamespacedName(const AZStd::string& ns, const AZStd::string& name)
+    {
+        AZStd::string namespacedName;
+        if (!ns.empty())
+        {
+            namespacedName = ns + "/";
```

Comment:
Seeing this always makes me check if there's any formatting tool available.
I found this: https://sourcegraph.com/search?q=context:o3de+lang:C%2B%2B+string::format&patternType=literal
API docs: https://www.o3de.org/docs/api/frameworks/azcore/class_a_z_std_1_1basic__string.html
It uses C syntax, so it's not as convenient as {fmt}, but better than gluing strings with concat operator.

Reply:
Nice remark, I will apply this